### PR TITLE
Whether a field belongs to the table

### PR DESCRIPTION
**Pull request by potaracom**

### CODE REVIEW

**Comment by toshi-toma** [comment]
related issue #10

File: src/getFields.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ export function createGetFields(
               .reduce((acc: any, cur: any) => acc.concat(cur), [])
           ];
         case "SUBTABLE":
+          layout.fields.forEach((layoutField: any) => (layoutField.tableCode = layout.code));
```
Comment (toshi-toma) [inline src/getFields.ts]: What do you think `subtableCode` instead of `tableCode`?
https://github.com/kintone/config-helper#field-types

Comment (potaracom) [reply src/getFields.ts]: That's a good idea 👍

**Comment by toshi-toma** [comment]
@potaracom 
I fixed just a nit or two.
Thank you for fix this issue!!

**Pull request merged**